clients/broadcaster_remote.go

```diff
@@ -107,9 +107,13 @@ func CreateStream(c Credentials, streamName string, profiles []video.EncodedProf
 	if err != nil {
 		return "", fmt.Errorf("appending stream to api url %s: %v", c.CustomAPIURL, err)
 	}
-	// prepare payload
-	payload := createStreamPayload{Name: streamName}
-	payload.Profiles = append(payload.Profiles, profiles...)
+	if err := validateProfiles(profiles); err != nil {
```

Comment:
Yeah, we will not send any profile. This is to catch coding errors, as copy profiles should never be sent to the broadcaster. If they are sent, this is a mistake like a "null reference exception". I think it would be misleading to make the filter here, as it could lead to behavioral bugs instead which might be harder to catch than if it consistently failed, for example. 

I could make this even more explicit by turning into a `panic` instead, but I think that would be too much. WDYT?

Reply:
Will add unit test